app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack marginTop={-10} marginBottom={12} alignItems={'flex-end'}>
+        <Heading size="md" fontWeight="bold" marginRight={-5}>
+          My Experience
```

Comment:
If I'm looking at the correct design I think this needs to be the `BebasNeue-Regular` font. I recall we added it together when we did the code fly through. Is there a reason you removed it from `StaTheme`?

Reply:
@PammyLo i lost the changes we made when i switched from the main branch. I have tried incorporating the above font but no luck yet getting it working. Obviously i am doing somethign wrong . WIll have another go later on this afternoon.

Reply:
If you pull the new changes into main and merge into your branch then the theme will be up to date. You could then use fontFamily=“title” here and you won’t need fontWeight.

It does work this way but the idea is that the fonts are set in the theme, so that should they change in the future it only needs to be updated in the theme.

Reply:
Thanks for that Pamela! I will pull the branch shortly and merge your changes and then hopefully have the font from the theme like you suggest.